library/murmurhash.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * 2015+ Copyright (c) Kirill Smorodinnikov <shaitkir@gmail.com>
+ * All rights reserved.
```

Comment:
This whole code was actually copy-pasted by Alexey Ivanov from Austin Appleby's original code at https://sites.google.com/site/murmurhash/